### DISCUSSION TITLE
Search r-universe from R

### DISCUSSION THREAD

**Discussion by robjhyndman** [r-universe-org/help#236] [General]
`pkgsearch::ps` hasn't been working for months, and I wonder if `r-universe` search might provide a replacement. Does https://r-universe.dev/search/ have an API that can be used from within R? Or can the results of a search be downloaded as a data frame?

**Comment by jeroen** [comment]
Yes of course, you can pass the query from the UI to https://r-universe.dev/stats/powersearch

For example: https://r-universe.dev/search/?q=jeroen%20ooms is https://r-universe.dev/stats/powersearch?limit=50&all=true&q=jeroen%20ooms

Note that you need to url-escape the `q` for example a space is "%20".

**Reply by robjhyndman** [reply]
Thanks
